CODEOWNERS

```diff
@@ -0,0 +1,10 @@
+boot/boot_serial @nordicjm @de-nordic
+boot/bootutil @davidvincze
+boot/cypress @romanjoe
+boot/espressif @almir-okato
+boot/mbed @AGlass0fMilk
+boot/mynewt @kasjer
+boot/muttx @michallenc
```

Comment:
Presumably, this should be `boot/nuttx` with an 'n'.

Reply:
Fixed